src/controllers/userController.js

```diff
@@ -18,47 +18,49 @@ const register = async (req, res) => {
         
         res.status(StatusCodes.CREATED).json({ userData, token });
     } catch (error) {
-        res.status(StatusCodes.BAD_REQUEST).json({ error: error.message });
+      console.error(error);
+      res.status(StatusCodes.BAD_REQUEST).json({ error: error.message });
```

Comment:
we are handling errors in different ways can we make everyhting the same? what we ahve: 
res.status(StatusCodes.BAD_REQUEST).json({ error: error.message });
res.status(statusCode).json({ message: errorMessage });
 .status(StatusCodes.INTERNAL_SERVER_ERROR)
      .json({ message: 'Logout failed' });
      
      I'll still approve, just maybe clean this up.

Reply:
Thank you I'll fix it!